sample/client.c

```diff
@@ -204,29 +208,42 @@ int main(int argc, char **argv)
   assert(mozquic_unstable_api1(&config, "tolerateNoTransportParams", 1, 0) == MOZQUIC_OK);
   assert(mozquic_unstable_api1(&config, "maxSizeAllowed", 1470, 0) == MOZQUIC_OK);
 
-  mozquic_new_connection(&c, &config);
-  mozquic_set_event_callback(c, connEventCB);
-  mozquic_start_client(c);
-
-  uint32_t i=0;
-  do {
-    usleep (1000); // this is for handleio todo
-    uint32_t code = mozquic_IO(c);
-    if (code != MOZQUIC_OK) {
-      fprintf(stderr,"IO reported failure\n");
-      break;
-    }
-    if (_getCount == -1) {
-      break;
-    }
-  } while (++i < 2000 || _getCount);
+  int test0rtt = has_arg(argc, argv, "-0rtt", &argVal);
+  if (test0rtt) {
+    assert(mozquic_unstable_api1(&config, "enable0RTT", 1, 0) == MOZQUIC_OK);
+  }
 
-  if (has_arg(argc, argv, "-streamtest1", &argVal)) {
-    streamtest1(c);
+  int repeat = 1;
+  if (has_arg(argc, argv, "-connectionresume", &argVal) ||
```

Comment:
its not worth putting a lot of effort into, but can we have a connection routine (and event handler) to just do the handshake/close and then go into the normal test (rather than e.g. doing the get more than one time).